### PR TITLE
Add AWS-CLI and ImageMagick tools

### PR DESCRIPTION
**Pull request by SzymonMrozek**
Hey! 
In the beginning, I want to say huge thanks for making that repo public and having a second opportunity to contribute to it! 

I added two tools that one of my open-source projects is using. Not sure if it's a proper approach to introduce tools that the project uses to this template repo, but these two seem to be quite universal for other developers. Let me know what you think :)

### CODE REVIEW

File: templates/xcode.pkr.hcl [hunk 1]
```diff
@@ -127,7 +127,9 @@ build {
     inline = [
       "source ~/.zprofile",
       "brew install graphicsmagick",
+      "brew install imagemagick",
       "brew install wix/brew/applesimutils",
+      "curl 'https://awscli.amazonaws.com/AWSCLIV2.pkg' -o 'AWSCLIV2.pkg' && sudo installer -pkg AWSCLIV2.pkg -target /"
```
Comment (fkorotkov) [inline templates/xcode.pkr.hcl:132]: What do you think of using Homebrew and doing it in the `base` variant?

File: templates/xcode.pkr.hcl [hunk 2]
```diff
@@ -127,7 +127,9 @@ build {
     inline = [
       "source ~/.zprofile",
       "brew install graphicsmagick",
+      "brew install imagemagick",
```
Comment (fkorotkov) [inline templates/xcode.pkr.hcl:130]: Could you please move it a line above to group tools for image processing:

`brew install graphicsmagick imagemagick`

File: templates/xcode.pkr.hcl (hunk 1 above)
Comment (SzymonMrozek) [reply templates/xcode.pkr.hcl:132]: While I could add it to the base variant I’m afraid that there is no homebrew support for AWSCLI

Comment (fkorotkov) [reply templates/xcode.pkr.hcl:132]: Seems `brew install awscli` installs the same package. Here is a link to [the formula](https://formulae.brew.sh/formula/awscli).

Comment (SzymonMrozek) [reply templates/xcode.pkr.hcl:132]: Oh, cool, thanks

**Review by fkorotkov — APPROVED** [review]

**Comment by fkorotkov** [comment]
The following images now have this change:

```
ghcr.io/cirruslabs/macos-ventura-base:latest
ghcr.io/cirruslabs/macos-ventura-xcode:14.3.1
ghcr.io/cirruslabs/macos-ventura-xcode:15-beta
```

**Pull request merged**